insights/parsers/sys_kernel.py

```diff
@@ -0,0 +1,38 @@
+"""
+SysKernel - file ``/proc/sys/kernel`` or ``/sys/kernel/``
+=========================================================
+
+The parsers included in this module are:
+
+SchedRTRuntime - File ``/proc/sys/kernel/sched_rt_runtime_us``
+--------------------------------------------------------------
+"""
+
+from insights import Parser, parser, LegacyItemAccess, get_active_lines
+from insights.parsers import ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.sched_rt_runtime_us)
+class SchedRTRuntime(Parser, LegacyItemAccess):
```

Comment:
`LegacyItemAccess` is only useful for `ParserClass` which has a `data` attribute in `dict` type. Here in this parser it's useless.

BTW, The `LegacyItemAccess` going to be deprecated.